api/src/main/proto/input/reqreply/chat/create.proto

```diff
@@ -32,3 +32,7 @@ message ChatCreateResponse {
     }
   }
 }
+
+service ChatCreateService {
```

Comment:
it's better to have RPCs in separate files. e.g.  chat-endpoints where you'll have all CRUD RPCs for chat

Reply:
I'll separate them. 
Is there any conventions about naming this files? 
It's okay if I save it in proto/input/grpc/crud.proto ?